src/engine/Input/Keyboard.ts

```diff
@@ -208,4 +266,11 @@ export class Keyboard extends Class {
         return code;
     }
   }
+
+  /**
+   * Gets the key code from KeyboardEvent.code if supported otherwise from KeyboardEvent.keyCode
+   */
+  private _getKeyCode(ev: KeyboardEvent): number {
+    return ev.code !== undefined ? parseInt(ev.code, 10) : ev.keyCode;
```

Comment:
@rledford I don't think that `parseInt` will work on `ev.code` as expected.  `ev.code` is a string constant of the form "KeyA" 

![image](https://user-images.githubusercontent.com/612071/94995170-23c55180-0562-11eb-85c9-1d08d939f645.png)

See sandbox https://codesandbox.io/s/key-code-y6994?file=/src/index.ts

Reply:
We are dropping support for IE so we don't need to worry about the the case where `ev.keyCode` is present and `ev.code` is not. Any objections to that @jedeen @kamranayub?

Reply:
the keyboard events in the chromium browser in the karma tests only have keyCode and I didn’t want to push anything that didn’t pass. I’ll change it to only use the KeyboardEvemt.code though.

Reply:
We'll likely need to update some tests :+1: thanks!